clang/lib/Sema/SPIRVBuiltins.td

```diff
@@ -367,7 +367,7 @@ def IntLongFloatGenType1   : GenericType<"IntLongFloatGenType1", TLIntLongFloats
 
 // GenType definitions for every single base type (e.g. fp32 only).
 // Names are like: GenTypeFloatVecAndScalar.
-foreach Type = [Bool, Char, UChar, Short, UShort,
+foreach Type = [Bool, Char, UChar, SChar, Short, UShort,
```

Comment:
`Bool` should be removed from here

Reply:
Updated. Let me know if I missed anything.